ivadomed/transforms.py

```diff
@@ -308,3 +312,190 @@ def __call__(self, sample):
             sample.update(rdict)
 
         return sample
+
+### 3D Transforms ###
+
+
+class ElasticTransform3D(mt_transforms.ElasticTransform):
+    @staticmethod
+    def elastic_transform(image, alpha, sigma):
+        shape = image.shape
+        dx = gaussian_filter((np.random.rand(*shape) * 2 - 1),
+                             sigma, mode="constant", cval=0) * alpha
+        dy = gaussian_filter((np.random.rand(*shape) * 2 - 1),
+                             sigma, mode="constant", cval=0) * alpha
+        dz = gaussian_filter((np.random.rand(*shape) * 2 - 1),
+                             sigma, mode="constant", cval=0) * alpha
```

Comment:
If not too much work --> can we combine it with the 2D transform? I think there is not too much difference between the two implementations.
(it can also be done later, after the MIDL deadline, just opening an issue as reminder :) )